### PR TITLE
🌐 Polish translations for 2.7.5

### PR DESCRIPTION
**Pull request by DudekArtur**
A Polish translation has been added based on the instance used in 2.7.3. Differences due to the current English version of the translation have been taken into account.

### CODE REVIEW

**Comment by piRGoif** [comment]
Thanks ! I will review this !
For reference #194 was added to integrate same translations but for the develop branch (next 3.0.0). This one is for 2.7.5 (2.7.4 is coming really close, so too late to integrate this PR in that version)

**Comment by piRGoif** [comment]
I've just renamed the PR so that it will be easier to us :)

File: dictionaries/pl.dictionary.itop.model.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?php
```
Comment (piRGoif) [inline dictionaries/pl.dictionary.itop.model.php]: This file doesn't exist in the EN version, can you remove it ?

**Review by piRGoif — CHANGES_REQUESTED** [review]
Wonderful job, thanks ! Just a little change to make : one empty file should be removed

File: dictionaries/pl.dictionary.itop.model.php (hunk 1 above)
Comment (DudekArtur) [reply dictionaries/pl.dictionary.itop.model.php]: Every other translation has an empty _**dictionaries/XX.dictionary.itop.model.php**_ file. That is why I also included an identical one in the Polish translation. It is not only in the English translation. I don't know what these files are for. Are you sure I should delete it?

Comment (piRGoif) [reply dictionaries/pl.dictionary.itop.model.php]: I am :)
After merging your PR I'll delete dictionnary.model.php files in other languages.

Comment (DudekArtur) [reply dictionaries/pl.dictionary.itop.model.php]: Done

**Review by piRGoif — APPROVED** [review]

**Comment by piRGoif** [comment]
This seems all good to me thanks !
This PR is on hold waiting for the 2.7.4 to be packaged and published... It should take a few weeks then we'll be able to merge for 2.7.5, and deal with #194.

**Comment by piRGoif** [comment]
I'm really sorry, as we had some troubles recently with bad dictionaries, the products team prefers not to integrate any new languages in the 2.7.* branch...
All my apologies, I should have checked with them earlier... 😢

However you could add those dictionaries in a dedicated GitHub repo, or maybe factorize them so they could be installed a additional modules ?

I'm closing this PR, again, all my apologies :(

**Pull request closed without merging**